src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep

```diff
@@ -307,11 +307,11 @@ var invalidInstanceFunctionAccess = a.b.c.bar()
 var invalidInstanceFunctionCall = az.az()
 //@[37:39) [BCP086 (Error)] The function "az" does not exist in namespace "az". |az|
 var invalidPropertyAccessOnAzNamespace = az.az
-//@[44:46) [BCP055 (Error)] Cannot access properties of type "namespace". An "object" type is required. |az|
+//@[41:43) [BCP063 (Error)] The name "az" is not a parameter or variable. |az|
```

Comment:
Should we change the error so it says `The name "{0}" is not a parameter or variable.`?